build/ndm-daemonset/entrypoint.sh

```diff
@@ -2,17 +2,28 @@
 
 export GOTRACEBACK=crash
 
+# openebs base directory inside the container
+OPENEBS_BASE_DIR="/var/openebs"
```

Comment:
Is this variable required?

Reply:
This variable helps to understand how the NDM base path is constructed using the openebs base directory inside the container. Other than that it provides no additional information and is not used.